data/com.github.Alcaro.Flips-symbolic.svg

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="16px" height="16px" viewBox="0 0 16 16" version="1.1">
+<g id="surface45">
+<path style=" stroke:none;fill-rule:nonzero;fill:rgb(14.117648%,12.156863%,19.215687%);fill-opacity:1;" d="M 3 0 L 3 2 L 4 2 L 4 3 L 5 3 L 5 5 L 6 5 L 6 8 L 5 8 L 5 9 L 3 9 L 3 10 L 0 10 L 0 12 L 1 12 L 1 13 L 2 13 L 2 14 L 3 14 L 3 15 L 5 15 L 5 16 L 9 16 L 9 15 L 11 15 L 11 14 L 12 14 L 12 13 L 13 13 L 13 12 L 14 12 L 14 10 L 15 10 L 15 6 L 14 6 L 14 4 L 13 4 L 13 3 L 12 3 L 12 2 L 11 2 L 11 1 L 9 1 L 9 0 Z M 8 5 L 9 5 L 9 8 L 8 8 Z M 10 5 L 11 5 L 11 8 L 10 8 Z M 6 9 L 7 9 L 7 11 L 10 11 L 10 10 L 11 10 L 11 9 L 12 9 L 12 10 L 11 10 L 11 11 L 10 11 L 10 12 L 7 12 L 7 11 L 6 11 Z M 6 9 "/>
+</g>
+</svg>
```

Comment:
It's fairly common at GNOME to have all the desktop-related metadata (icons, launcher, store info…) in a data/ dir, but really it's nothing more than an implicit convention and it could be moved elsewhere. Just ask and I'll move them. :)